packages/versions/src/lib/checkFuelCoreVersionCompatibility.ts

```diff
@@ -4,6 +4,12 @@ import { majorEq, minorEq, patchEq } from './semver';
 export function checkFuelCoreVersionCompatibility(networkVersion: string) {
   const { FUEL_CORE: supportedVersion } = getBuiltinVersions();
 
+  if (networkVersion.match(/^[0-9]+\.[0-9]+\.[0-9]+$/) === null) {
+    console.warn(
+      `You're running against an unreleased fuel-core version: ${networkVersion}. Things may work as expected, but it's not guaranteed. Please use a released version.`
+    );
```

Comment:
I believe the warning message should include something like (not the same sentence but something with the same meaning):

_This non-official `fuel-core` build may include features and updates not yet supported by this version of the TS-SDK._

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1855/commits/a519b9baeee95db92d9219cd165d8c0ad9562408